app/models/user.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class User
+
+  attr_reader :name, :job, :email, :address
```

Comment:
Same with the blank line, it should be between attr_readers and initialize method